cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -336,7 +336,7 @@ public void run() {
         }
         while (!sampleLoadingFinished(endOffsets)) {
           try {
-            ConsumerRecords<byte[], byte[]> consumerRecords = _consumer.poll(10);
+            ConsumerRecords<byte[], byte[]> consumerRecords = _consumer.poll(1000);
```

Comment:
I feel this does not needs to be configured by user.  since here we seek to a previous offset and consumes to the current end offset, which guarantees that there will always be data back, we only needs to set the poll time large enough to prevent thread from burning CPU cycle.

Reply:
I agree that the user should not configure it -- i.e. moving the hardcoded value to a variable is just for clarity.
```
private static final long SAMPLE_POLL_TIMEOUT = 1000L;
...
ConsumerRecords<byte[], byte[]> consumerRecords = _consumer.poll(SAMPLE_POLL_TIMEOUT);
```